src/mcp/shared/progress.py

```diff
@@ -77,7 +78,7 @@ def progress(
     if ctx.meta is None or ctx.meta.progressToken is None:
         raise ValueError("No progress token provided")
 
-    progress_ctx = ProgressContext(ctx.session, ctx.meta.progressToken, total)
+    progress_ctx = ProgressContext(ctx.session, ctx.meta.progressToken, total, None)
```

Comment:
Do we even need message in progress context?
```suggestion
    progress_ctx = ProgressContext(ctx.session, ctx.meta.progressToken, total)
```